src/daft-parquet/src/python.rs

```diff
@@ -124,15 +124,15 @@ pub mod pylib {
             )
         })?;
         let (schema, all_arrays, num_rows) = read_parquet_result;
-        let pyarrow = py.import(pyo3::intern!(py, "pyarrow"))?;
+        let pyarrow = py.import_bound("pyarrow")?;
         convert_pyarrow_parquet_read_result_into_py(py, schema, all_arrays, num_rows, pyarrow)
     }
     #[allow(clippy::too_many_arguments)]
     #[pyfunction]
     pub fn read_parquet_bulk(
         py: Python,
-        uris: Vec<&str>,
-        columns: Option<Vec<&str>>,
+        uris: Vec<String>,
```

Comment:
Why are we making this owned if we just take the ref anyways?

Reply:
pyo3 limitations. See comment on `read_parquet`